src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Action action;
+    private String fruit;
+    private int value;
+
+    public FruitTransaction(Action action, String fruit, int value) {
+        this.action = action;
+        this.fruit = fruit;
+        this.value = value;
+    }
+
+    public Action getAction() {
+        return action;
+    }
+
+    public void setType(Action action) {
+        this.action = action;
+    }
```

Comment:
The method name 'setType' does not accurately describe what the method does. It should be renamed to 'setAction' to match the field it is modifying.